src/filebrowser/dialogs.ts

```diff
@@ -291,6 +291,7 @@ class CreateFromHandler extends Widget {
           return widget;
         });
       }
+      this._model.deleteFile(this._orig);
```

Comment:
@blink1073  One need to call refresh on FileBrowserModel too. Otherwise the file will remain in file browser for a while. Or am I missing something?